src/components/Camera/Camera.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
```

Comment:
can we split this camera stuff into another PR? Same with the Audio stuff. Appreciate you making these though!